django/views/i18n.py

```diff
@@ -218,51 +215,15 @@ def get_javascript_catalog(locale, domain, packages):
         paths.append(path)
     # add the filesystem paths listed in the LOCALE_PATHS setting
     paths.extend(reversed(settings.LOCALE_PATHS))
```

Comment:
Excellent! Feel free to poke me once you have it published as I'm interested javascript_catalog and the translation code in general that feels like it is missing some love for such a powerful tool.